src/components/bc-modal.ts

```diff
@@ -14,9 +13,7 @@ export class Modal extends withTwind()(BitcoinConnectElement) {
       class="fixed top-0 left-0 w-full h-full flex justify-center items-end sm:items-center z-[21000]"
     >
       <div
-        class="absolute top-0 left-0 w-full h-full -z-10 ${classes[
-          'bg-foreground'
-        ]} animate-darken"
+        class="absolute top-0 left-0 w-full h-full -z-10 bg-black animate-darken"
```

Comment:
black seems to be hardcoded now - could you undo the change and instead add `opacity-80` for some transparency which should have the same effect as bg-black/80 which you mentioned in the issue

Reply:
`bg-foreground` is white in dark mode, it's not an issue of transparency.

Reply:
Ah, it's on the darken animation, which darkens to 50% opacity. Thanks!